lib/diagnostic.js

```diff
@@ -17,7 +17,36 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+const car = {
+  make: 'Toyota',
+  model: 'Prius',
+  year: 2016,
+  drive: function () {
+    let go = 0;
+    if (go < 1) {
+      return 'Drive';
+    }
+  },
+  brake: function () {
+    let stop = 1;
+    if (stop === 1) {
+      return 'Hold On';
+    }
+  },
+  park: function (){
+
+  },
+};
+const salesPerson = {
+  name: 'Louis Breguet',
+  commissionRate: 0.15,
+  sell: function (price) {
+    return price * this.commissionRate;
+  },
+  call: function () {
+
+  },
+};
```

Comment:
We're just looking for two examples of entities, along with their attributes and methods, that can be used in a browser-based calendar application.